### PR TITLE
feat: Add docs: diagram, costs

### PR DESCRIPTION
**Pull request by pamelaluquemck**
Add new Azure diagram, .md with costs.

### CODE REVIEW

File: docs/azure_architecture.png
Comment (pramirezma) [inline docs/azure_architecture.png:1]: QQ: Why is the Azure Container Registry so alone outside of azure?

File: docs/azure_architecture_costs.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Costo por mes durante 24 horas:
+
+* Virtual Network: 4 USD
+* Container Instance: 32.81 USD  - 1 Container group: 2 instances
+* CosmosDB for MongoDB: 23.36 USD - Standard provisioned throughput
+* Load Balancer: 0 USD
+* Azure Container Registry: 5 USD - Basic Tier
+
+**Total cost: 272,83 USD**
```
Comment (jose-sal-y-rosas) [inline docs/azure_architecture_costs.md:23]: I think summarizing it is not 272.83, something is missing?

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please check total cost

**Review by pramirezma — CHANGES_REQUESTED** [review]
Check if the diagram is correct

File: docs/azure_architecture.png
Comment (pamelaluquemck) [reply docs/azure_architecture.png:1]: Sorry, I will put into the resource group

File: docs/azure_architecture_costs.md (hunk 1 above)
Comment (pamelaluquemck) [reply docs/azure_architecture_costs.md]: My mistake, I will update the total cost

**Review by pramirezma — APPROVED** [review]

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Pull request merged**